awspub/image.py

```diff
@@ -502,6 +457,53 @@ def create(self) -> Dict[str, _ImageInfo]:
 
         return images
 
+    def _register_images(self, snapshot_ids: Dict[str, str], region: str, ec2client_region: EC2Client):
```

Comment:
* function documentation missing (including return value). see other functions
* the function does register a single image, so use `_register_image` as name
* there's no need to pass all snapshot_ids into the function. it just needs the snapshot id for the region.
* the region parameter is then no longer required.
*

Reply:
i'll make the first three changes but believe region should still be passed for logging purposes. i could move the logging outside for function, but i prefer to avoid the temporal coupling.